frontend/components/WindowFolder/WindowDefinitions.tsx

```diff
@@ -196,14 +182,14 @@ export default function WindowDefinitions(windowState) {
       component: (w, id, color) => <Clusters id={id} windata={w} color={color} />,
     },
     Cluster: {
```

Comment:
if it's the same as group, should it just be group?

Reply:
Yeah better yet it doesn't need to exist anymore.